code/week5/week5.h

```diff
@@ -0,0 +1,9 @@
+#pragma once
```

Comment:
No, I meant, why does week5.h exist as a file? The only place I see it being included is in main.cpp, so this list of headers should just live directly in main.cpp, with no need for this extra file.